infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -114,59 +111,21 @@ public class LogReplicationSinkManager implements DataReceiver {
      * Constructor Sink Manager
      *
      * @param metadataManager manages log replication session's metadata
-     * @param serverContext server level context
      * @param session log replication session unique identifier
      * @param replicationContext log replication context
      */
-    public LogReplicationSinkManager(LogReplicationMetadataManager metadataManager,
-                                     ServerContext serverContext, LogReplicationSession session,
+    public LogReplicationSinkManager(LogReplicationMetadataManager metadataManager, LogReplicationSession session,
                                      LogReplicationContext replicationContext) {
 
         this.replicationContext = replicationContext;
-        this.runtime = CorfuRuntime.fromParameters(CorfuRuntime.CorfuRuntimeParameters.builder()
-                .trustStore((String) serverContext.getServerConfig().get(ConfigParamNames.TRUST_STORE))
-                .tsPasswordFile((String) serverContext.getServerConfig().get(ConfigParamNames.TRUST_STORE_PASS_FILE))
-                .keyStore((String) serverContext.getServerConfig().get(ConfigParamNames.KEY_STORE))
-                .ksPasswordFile((String) serverContext.getServerConfig().get(ConfigParamNames.KEY_STORE_PASS_FILE))
-                .tlsEnabled((Boolean) serverContext.getServerConfig().get("--enable-tls"))
-                .maxCacheEntries(replicationContext.getConfig(session).getMaxCacheSize())
```

Comment:
the max cache entries in the runtime created in InterClusterReplicationServer is half.  Something to consider during performance evaluation